commd_test.go

```diff
@@ -26,7 +26,7 @@ func TestGenerateUnsealedCID(t *testing.T) {
 
 	expCommD := cidMustParse("baga6ea4seaqiw3gbmstmexb7sqwkc5r23o3i7zcyx5kr76pfobpykes3af62kca")
 
-	commD, _ := commp.PieceAggregateCommP(
+	commD, _, _ := commp.PieceAggregateCommP(
```

Comment:
I think you could work out the expected value here and add an assertion for it so we at least have some coverage